package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "new.expensify",
-  "version": "1.1.70-1",
+  "version": "1.1.70-3",
```

Comment:
idk what is up with GitHub today, but this is the version on main ... :rage1: 

https://raw.githubusercontent.com/Expensify/App/main/package-lock.json
![2022-06-01_15-22-27](https://user-images.githubusercontent.com/32969087/171527405-79c768a4-4836-4e61-b120-edb8cf5cf48f.png)